flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/KubernetesOperatorConfigOptions.java

```diff
@@ -365,22 +365,45 @@ public static String operatorConfigKey(String key) {
                     .withDescription(
                             "Initial interval of automatic reconcile retries on recoverable errors.");
 
+    @Documentation.Section(SECTION_SYSTEM)
+    public static final ConfigOption<Duration> OPERATOR_RETRY_MAX_INTERVAL =
+            operatorConfig("retry.max.interval")
+                    .durationType()
+                    .defaultValue(Duration.ZERO)
+                    .withDescription(
+                            "Max interval of automatic reconcile retries on recoverable errors, if set to <=0, no limit is applied.");
+
     @Documentation.Section(SECTION_SYSTEM)
     public static final ConfigOption<Double> OPERATOR_RETRY_INTERVAL_MULTIPLIER =
             operatorConfig("retry.interval.multiplier")
                     .doubleType()
-                    .defaultValue(2.0)
+                    .defaultValue(1.5)
```

Comment:
Why are we changing this value here? It's unrelated to the rate limiting

Reply:
Based on some testing I felt that the current 2 multiplier for the exponential backoff was a bit too aggressive in some cases. So 1.5 with a little longer max retries felt more appropriate. If you feel we shouldn't change this I remove it from the PR